clients/src/main/java/org/apache/kafka/common/requests/ListAclsResponse.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.common.requests;
+
+import org.apache.kafka.common.protocol.ApiKeys;
+import org.apache.kafka.common.protocol.Errors;
+import org.apache.kafka.common.protocol.types.Struct;
+
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ListAclsResponse extends AbstractResponse {
+    private final static String THROTTLE_TIME_MS = "throttle_time_ms";
+    private final static String ERROR_CODE = "error_code";
+    private final static String ERROR_MESSAGE = "error_message";
+    private final static String RESOURCES = "resources";
+    private final static String ACLS = "acls";
+
+    private final int throttleTimeMs;
+    private final Throwable throwable;
+    private final Collection<AclFixture> acls;
+
+    public ListAclsResponse(int throttleTimeMs, Throwable throwable, Collection<AclFixture> acls) {
+        this.throttleTimeMs = throttleTimeMs;
+        this.throwable = throwable;
+        this.acls = acls;
+    }
+
+    public ListAclsResponse(Struct struct) {
+        this.throttleTimeMs = struct.getInt(THROTTLE_TIME_MS);
+        short errorCode = struct.getShort(ERROR_CODE);
+        String errorMessage = struct.getString(ERROR_MESSAGE);
+        if (errorCode != 0) {
```

Comment:
Nit: we typically do the `Errors.forCode` inline with the `getShort` and then check against `NONE` instead of `0`